plugins/renovate-backend/src/wrapper/utils.ts

```diff
@@ -93,3 +93,16 @@ export function getCacheEnvs(
     RENOVATE_REDIS_URL: connection,
   };
 }
+
+export function getPassthroughEnvs(config: Config): Record<string, string> {
+  const env: Record<string, string> = {};
+  const passthroughEnvs = getPluginConfig(config).getOptionalConfigArray('runtime.environment') ?? [];
+
+  passthroughEnvs.forEach(e => {
+    const name = e.getString('name');
+    const value = e.getOptionalString('value') ?? undefined;
+    env[name] = value ?? process.env[name] ?? '';
```

Comment:
```suggestion
    const value = e.getOptionalString('value') ?? process.env[name];
    if (value) {
      env[name] = value;
    }
```
Simplify and do not set the variable if there is no value.

Reply:
Done, also added a debug log if it's not mapped.